packages/storybook/stories/ControlBar/Buttons/Participants/Participants.stories.tsx

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ParticipantsButton, ParticipantListProps } from '@azure/communication-react';
+import { Canvas, Description, Heading, Props, Source, Title } from '@storybook/addon-docs/blocks';
+import { boolean, text } from '@storybook/addon-knobs';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+
+import { COMPONENT_FOLDER_PREFIX } from '../../../constants';
+// import { OptionsButtonCustomExample } from './snippets/Custom.snippet';
```

Comment:
Did you accidentally leave in this commented import and the one on line 20?

Reply:
oopsi... forgot to remove those comments :)